mutation_partition.cc

```diff
@@ -250,15 +250,33 @@ void mutation_partition::ensure_last_dummy(const schema& s) {
 }
 
 void mutation_partition::apply(const schema& s, const mutation_partition& p, const schema& p_schema) {
-    apply_weak(s, p, p_schema);
+    mutation_application_stats app_stats;
+    return apply(s, p, p_schema, app_stats);
```

Comment:
Why did you change apply_weak() to apply()?

Reply:
Ah, I understand the applyapply_weak thing. But leaving a version that doesn't accept a stats object is an invitation to forget to propagate them. It's very annoying to always have to propagate these objects, but also very safe.